### ISSUE TITLE
How do I derive weights for CrossEntropy Loss on my custom dataset?

### ISSUE THREAD

**Issue by sarmientoj24** [pytorch/pytorch#39993]
I have five classes + 1 background for my semantic segmentation. My mask is like the one below:
It has a ton of background and classes are rarely seen on both image. 
![download (2)](https://user-images.githubusercontent.com/8830319/84572938-10825080-add0-11ea-9285-0082814edb94.png)

I've seen codes using CrossEntropyLoss with designated weights but usually they are using available datasets such as CityScapes so they have the weights on them.

How do I derive CrossEntropyLoss weights for my dataset?
